assets/webpack.config.js

```diff
@@ -40,16 +39,10 @@ module.exports = (env, options) => ({
           test: /\.css$/,
           use:  [MiniCssExtractPlugin.loader, 'css-loader']
       },
-      {
-        test: /\.(png|woff|woff2|eot|ttf|svg)$/,
```

Comment:
Technically, we don't have nor use such files. Though, Webpack stubbornly carries Bootstrap font `Glyphicons Halflings` to `./priv/static/assets` regardless of this step :| 

But again, seems that the codebase doesn't use glyphicons. So after moving to ESBuild we can ignore those files.